metricbeat/tests/system/test_processors.py

```diff
@@ -60,21 +63,21 @@ def test_dropfields_with_condition(self):
         )
         metricbeat = self.start_beat()
         self.wait_until(
-            lambda: self.output_count(lambda x: x >= 1),
+            lambda: self.output_count(lambda x: x >= 2),
```

Comment:
Why is the number of expected events increased?

Reply:
So, the test checks the CPU percentage metrics, which won't exist until the second event, as it's calculated from the delta of two events. This CPU calculation hasn't changed, but what has changed is that we're no longer reporting metrics that don't exist, so the very first event won't have CPU percentages.